ui/main/src/app/views/customCardList/resultTable/ResultTableExport.ts

```diff
@@ -67,6 +67,10 @@ export class ResultTableExport {
                             cellValue = cellValue?.rowValue;
                             row[column.headerName] = cellValue;
                             break;
+                        case 'select':
+                            cellValue = cellValue?.value;
+                            row[column.headerName] = cellValue;
+                            break;
```

Comment:
We need to have the label if possible , so the idea is to iterate throught possible values to get label if exist .

The logic is already implemented in SelectCellRenderComponent : 


```
        this.fieldLabel =
            params.getValue()?.possibleValues?.find((value: any) => value.value === this.fieldValue)?.label ??
            this.fieldValue;
```
This solution will duplicate the logic , i guess it would be better when generating the result table to add a field text. So  the logic would be to implement it in TableRowBuilder (Line 128) 

So as it is will be compute when building the result table : 
-  it will not be needed to be compute in the SelectCellRenderComponent
-  for the export it will just be getting cellValue?.text as for default cell